setup.pl

```diff
@@ -894,8 +905,18 @@
 		if ($upgrading) {
 			$action = 'restart';
 		}
+		my $start_cmd_extra;
+		if ($upgrading && $killmodenonepl == 1) {
+			$start_cmd_extra = "$config_directory/.reload-init-systemd";
+			if (-r $start_cmd_extra) {
+				$start_cmd .= " ; $start_cmd_extra";
+				$start_cmd_extra = -1;
+				}
+			}
 		print "Attempting to $action Webmin web server ..\n";
 		$ex = system($start_cmd);
+		unlink($start_cmd_extra)
+			if ($start_cmd_extra == -1);
```

Comment:
How does unlinking a file named -1 work?

Reply:
I just used the same variable name to set it to -1. It will be set to -1 only if related code is used. (which is set just a few lines above). It could be anything in place of -1.

Reply:
Duh, you're right!! I just realized that I unset the actual file name! I will fix that!

Reply:
Fixed.